kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -4327,12 +4327,12 @@ messages:
          string = psPlayerDescription;
       }
       
-      if StringContain(psPlayerDescription,"�")
+      if StringContain(psPlayerDescription,"�")
```

Comment:
These spurious changes are probably due to different file encodings (probably ASCII vs. UTF-8), but in any case I don't think they're safe.

Reply:
I will try and fix this.  I'm noticing this happens with user.kod for me in other test branches.

Reply:
This is also fixed due to removal in commit d2c5198, resolving